python/cudf/cudf/core/column/column.py

```diff
@@ -1380,7 +1378,7 @@ def _return_sentinel_column():
         # values of `self`:
         order = arange(len(self))
         order = order.take(left_gather_map, check_bounds=False).argsort()
-        codes = codes.take(order)
+        codes = codes.take(order, check_bounds=False)
```

Comment:
```suggestion
        codes = codes.take(order)
```

Sorry, I didn't even check if `take` supports `check_bounds` (which it doesn't)...

`tests/test_dataframe.py::test_dataframe_init_from_series_list[None-data5-False] - TypeError: take() got an unexpected keyword argument 'check_bounds'`